uppsrc/Core/SSL/AES.cpp

```diff
@@ -223,7 +223,8 @@ bool Aes256Gcm::Decrypt(Stream& in, const String& password, Stream& out)
 			throw Exc("Decryption aborted");
 		
 		while(remaining > 0) {
-			String chunk = in.Get(min((int64) chunksize, remaining));
+			int64 csz = min((int64) chunksize, remaining);

```

Comment:
Replace (int64) with static_cast<int64>() for consistency with the second line.

Reply:
Done